lib/cbus_elixir/app/app.ex

```diff
@@ -0,0 +1,104 @@
+defmodule CbusElixir.App do
+  @moduledoc """
+  The App context.
+  """
+
```

Comment:
This is the issue with re-generating a resource with the context; it clobbers any existing context with the same name. So if we use `App` (which I believe is correct) we need to make sure to append the generated changes with the existing code in `app.ex`

Reply:
I branched from master last night so none of your stuff was ever in there.

Reply:
Sounds like a rebase off of master is needed now that it is in there?